frontend-react/src/content/resources/api-programmers-guide/documentation/data-model/DataModel.mdx

```diff
@@ -0,0 +1,70 @@
+import { Button, Icon, Alert, Table } from "@trussworks/react-uswds";
+import PatientDataElementsTable from "./PatientDataElementsTable.mdx";
+import OrderAndResultDataElementsTable from "./OrderAndResultDataElementsTable.mdx";
+import SpecimenDataElementsTable from "./SpecimenDataElementsTable.mdx";
+import OrderingProviderDataElementsTable from "./OrderingProviderDataElementsTable.mdx";
+import TestingFacilityDataElementsTable from "./TestingFacilityDataElementsTable.mdx";
+import AskOnEntryTable from "./AskOnEntryTable.mdx";
+import ReportingAndOrderingFacilityDataElementsTable from "./ReportingAndOrderingFacilityDataElementsTable.mdx";
+
+# Data model
+
+<p className="usa-intro text-base">View API CSV and HL7 Field Requirements in the tables on this page or download all the requirements as CSV.</p>
+<a href="https://docs.google.com/spreadsheets/d/1sSpDeKQdackzRS5IT58WmUWn1eq6pA1Rm57u0C5QpgM/edit?usp=sharing" className="usa-button">Download Excel file (.xlsx) <Icon.FileDownload /></a> Last updated: 1/6/23
+
+***
+
+## Common errors
+* **Two of the most important and often overlooked pieces of required data are the deviceIdentifier (OBX-17.1) and testPerformed (OBX-3.1).**
+These fields must match exactly to the appropriate row in the LOINC In Vitro Diagnostic (LIVD) test code mapping.
+[This is the most updated LIVD mapping](https://www.cdc.gov/csels/dls/sars-cov-2-livd-codes.html) from the CDC. Specifics about each field are detailed in the tables below.
+* **The preferred timestamp formatting for CSV and HL7 is yyyyMMddhhmmss+/-zzzz.**
+If the UTC offset (+/-zzzz) is not present, results should be normalized to a single time zone that’s agreed upon during the onboarding process.
+
+***
+
+## Legend
+
+* **Yes:** Required field for acceptance
+* **Yes** – Conditional: Required, but only under certain circumstances. Review the field’s Data Requirements and Additional Guidance for more information.
```

Comment:
Is this suppose to have a dash? – Conditional

Reply:
Yes but thanks for bringing this line up as the bolding was not correctly applied to the entire label.